src/gsy_e/models/strategy/market_agents/two_sided_agent.py

```diff
@@ -55,6 +55,11 @@ def usable_bid(self, bid: "Bid") -> bool:
         """Prevent MAEngines from trading their counterpart's bids."""
         return all(bid.id not in engine.forwarded_bids.keys() for engine in self.engines)
 
+    # pylint: disable=unused-argument
+    def event_bid(self, *, market_id: str, bid: "Bid"):
```

Comment:
Why do we need this interface in the first place? The market_id is not used here and also not in the engine. Same applies to the event_offer in the one-sided market.

Reply:
Indeed, the reason why I added this was in case these methods are used by the strategies in the future. In this case, there needs to be a way to distinguish which market emitted the offer / bid event, in order to treat differently future markets and spot markets.